matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/Event.kt

```diff
@@ -228,11 +229,14 @@ data class Event(
         return when {
             isReplyRenderedInThread() || isQuote() -> ContentUtils.extractUsefulTextFromReply(text)
             isFileMessage() -> "sent a file."
+            isVoiceMessage() -> "sent a voice message."
             isAudioMessage() -> "sent an audio file."
             isImageMessage() -> "sent an image."
             isVideoMessage() -> "sent a video."
-            isSticker() -> "sent a sticker"
+            isSticker() -> "sent a sticker."
             isPoll() -> getPollQuestion() ?: "created a poll."
+            isLiveLocation() -> "Live location."
```

Comment:
This seems to be a noun whereas all the others are actions? Maybe this could be made similar to the message below?

Reply:
Yes, I know. I aligned the string to what we have on other platforms and what was required by the design team. If we want to change, we would need to discuss with design team and update other platforms as well.

Reply:
Oh ok. In that case, let's leave it as is for now and not block your PR on this. Would you mind logging a short ticket about this in PSG-829? Then we can discuss whether or not to change this separately.

Reply:
Yes done.